src/sentry/integrations/slack/message_builder/base/block.py

```diff
@@ -58,10 +94,30 @@ class SlackBlockType(TypedDict):
 
         return action_block
 
+    @staticmethod
+    def get_context_block(text: str, timestamp: Optional[datetime] = None) -> SlackBlock:
+        if timestamp:
+            time = "<!date^{:.0f}^ {} at {} | Sentry Issue>".format(
+                to_timestamp(timestamp), "{date_pretty}", "{time}"
```

Comment:
```suggestion
            time = "<!date^{:.0f}^{} at {} | Sentry Issue>".format(
                to_timestamp(timestamp), "{date_pretty}", "{time}"
```
i believe the removal of the single space capitalizes lol